<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignValidate</class>
 <widget class="QWidget" name="SignValidate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmar - Validar Documentos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(76, 118, 82);
background-color:rgb(216, 230, 225);</string>
  </property>
  <layout class="QGridLayout" name="signValidateLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="titleLabel">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold;</string>
     </property>
     <property name="text">
      <string>Firmar - Validar Documentos</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>288</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="scrollAreaWidgetLayout">
       <item row="1" column="1">
        <widget class="QFrame" name="fileFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="fileFrameLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: bold;
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Archivo</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>3</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="3">
           <widget class="QLabel" name="label_razon">
            <property name="toolTip">
             <string>Solo en firmas de documentos PDF</string>
            </property>
            <property name="text">
             <string>Raz√≥n*:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="4">
           <widget class="QLineEdit" name="razon"/>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="browseFiles">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(11, 35, 21);
background-color: rgb(229, 229, 229);</string>
            </property>
            <property name="text">
             <string>Archivo</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QPushButton" name="validate">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(11, 35, 21);
background-color: rgb(229, 229, 229);</string>
            </property>
            <property name="text">
             <string>Validar</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/validate.png</normaloff>:/images/validate.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="sign">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(11, 35, 21);
background-color: rgb(229, 229, 229);</string>
            </property>
            <property name="text">
             <string>Firmar</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/sign.png</normaloff>:/images/sign.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0" rowspan="3" colspan="2">
           <widget class="SignListAreaWidget" name="filesWidget">
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="4">
           <widget class="QTextEdit" name="resumen"/>
          </item>
          <item row="8" column="3">
           <widget class="QLabel" name="label_lugar">
            <property name="toolTip">
             <string>Solo en firmas de documentos PDF</string>
            </property>
            <property name="text">
             <string>Lugar*:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="4">
           <widget class="QLineEdit" name="lugar"/>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_resumen">
            <property name="text">
             <string>Resumen:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QProgressBar" name="signValidateProgressBar">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
     <property name="format">
      <string>Procesando...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignListAreaWidget</class>
   <extends>QListWidget</extends>
   <header>client_fva/ui/custom_components/ownsignlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
